### DISCUSSION TITLE
Welcome to vscode-avdmanager Discussions!

### DISCUSSION THREAD

**Discussion by toroxx** [toroxx/vscode-avdmanager#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manuel-plavsic** [comment]
Hi, my name is Manuel and I am a software engineer and I am developing an E2EE app with flutter in my free time. I have just discovered your extension and it helped me a lot. For some reason, the `advancedFeatures.ini` always gets ignored on my machine, so I couldn't get `-feature -Vulkan` to be passed automatically on emulator startup within VSCode.  Your Emulator Opt setting is a lifesafer! Also, the UI is great. Thanks for your contribution.
